01-main/packages/gitkraken

```diff
@@ -1,9 +1,10 @@
 DEFVER=1
-get_website "https://help.gitkraken.com/gitkraken-client/current/"
+ARCHS_SUPPORTED="amd64"
+get_website "https://www.gitkraken.com/download/linux-deb"
 if [ "${ACTION}" != "prettylist" ]; then
-    VERSION_PUBLISHED=$(grep -m 1 'id="version-' "${CACHE_FILE}" | sed -e 's/<[^>]*>//g' | cut -d' ' -f2)
+    VERSION_PUBLISHED=$(sed -n -E "0,/.*name:\s*'([^']+)'/s//\1/p" "${CACHE_FILE}")
```

Comment:
This results in a <CR> at the end of the published version,which upsets `dpkg --compare-versions` when you update.

``` shell
 deb-get show gitkraken| cat -v -
[...]
  [^[[32m+^[[0m] Including local package gitkraken
[...]  
GitKraken
  Package:	gitkraken
  Repository:	99-local
  Updater:	deb-get
  Installed:	10.6.1
  Published:	10.6.1^M
```

Reply:
@philclifford Sorry, I'm not sure what, exactly, the error is. Is there a Ctrl-M at the end of the version?

I tested it, and it installed fine on Ubuntu 24.04, using the above command. I will retest.

Reply:
Yes, sorry to be unclear (at 4am ;-) ) - it is a Ctrl-M at the end of the version.  That only really affects update (because `dpkg --compare-versions` complains ) and probably upgrade (which will continually want to  upgrade because the compare "fails" ). Install will work fine as it just fetches the deb and installs it without comparing (except if the deb turns out to be the same version it'll skip the install )